invenio_app_rdm/records_ui/views/filters.py

```diff
@@ -121,3 +122,31 @@ def get_scheme_label(scheme):
     scheme_to_label = current_app.config.get("RDM_RECORDS_IDENTIFIERS_SCHEMES", {})
 
     return scheme_to_label.get(scheme, {}).get("label", scheme)
+
+
+def localize_number(value):
+    """Format number according to locale value."""
+    locale_value = current_app.config.get("BABEL_DEFAULT_LOCALE")
```

Comment:
note to self: this is always defined (has a default, https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/config.py#L122)